docs/build/quickstarts/connecting_to_the_world.md

```diff
@@ -53,319 +41,178 @@ This clearly is a much more minimalist scaffold than what the *quickstart* templ
+        }
+    } else {
+        result.error
+    }
 }
 ```
 
-That's right, just a few lines are required to bring HTTP requests to Wasm thanks to the heavy lifting done by Marine. The key to this module is the import of the [MountedBinaryResult](https://fluence.dev/docs/old-build/aquamarine/marine/marine-rs-sdk#mountedbinaryresult) at very top and the `extern` keyword at the bottom of the file. As mentioned earlier, *MountedBinary* allows the module to interact with the host's *curl* binary, whereas the [extern}(https://doc.rust-lang.org/std/keyword.extern.html) key word provides for a FFI block declaration for foreign code wrapped in the *marine* macro making it callable and linkable to other modules.
-
-Go ahead and delete the content in the `<...>/modules/curl_adapter/src/main.rs`and replace it with above code. Moreover, open the *module.yaml* file and update it to:
-
-```yaml
-version: 0
-
-type: rust
-
-name: curl_adapter
-mountedBinaries:            // add this and the line below, which tells the Marine runtime where to look for the binary
-  curl: /usr/bin/curl
-```
-
-We appended the location of the binary we want to call, in this case our curl binary, from the Wasm module as a MountedBinary. Make sure the path provided is correct, e.g. use `which curl` in your terminal and copy the resulting path if different than the one provided in the example code. Note that since we are using the REPL, all paths are to the local system. Make sure to make any necessary adjustments when deploying to remote.
-
-Now we just need update the service.yaml file so the service knows that the curl_adapter module exists:
+While the `demo_request` function is straight-forward, the `vault_path` function may need some explaining. To get some insight, see [here](https://github.com/fluencelabs/effectors/blob/e8e3251a80e3a44fc1887f7e7a47aef3c0ecfdcb/example/src/services/myRPC/modules/myRPC/src/main.rs#L48) and [vault](https://fluence.dev/docs/build/glossary#particle-file-vault) for more info. But basically, the effector module needs to know where it should look for the pertinent IO data realted to the underlying [particle](https://fluence.dev/docs/build/glossary#particle). This function uses the existing vault path and appends an effector specific dir.
 
-```yaml
-modules:
-  facade:
-    get: modules/hello_from
-  curl-adapter:                 # add this and the next line
-    get: modules/curl_adapter
-```
-
-Without worrying about our facade module, i.e., the module we want to use the curl-adapter from, go ahead and build the project with `fluence build` and fire up the REPL with `fluence service repl http_requester` and use the `i` or `interface` command to display the public interfaces of all loaded modules:
+Compile the code and let's have a look at it in the Marine REPL with `fluence service repl http-enabled`. Bur before you do, make sure you have the following directory in `.fluence/tmp/volumes/id-token`; if it's not there, create it. Back to the repl:
```

Comment:
```suggestion
Compile the code and let's have a look at it in the Marine REPL with `fluence service repl http-enabled`. But before you do, make sure you have the following directory in `http-enabled/.fluence/tmp/volumes/particles/id-token/curl_effector_dir`; if it's not there, create it:

```shell
# from the 'http-enabled' project root:
mkdir -p .fluence/tmp/volumes/particles/id-token/curl_effector_dir
```

Back to the repl:
```